README.md

```diff
@@ -6,3 +6,22 @@ closed.
 This package is disabled by default and can be enabled via the
 `autosave.enabled` config setting or from the Autosave section of the Settings
 view (`cmd-,`).
+
+## Service API
+
+### package.json
+``` json
+"consumedServices": {
+  "autosave": {
+    "versions": {
+      "0.22.0": "consumeAutosave"
+    }
+  }
+}
+```
+
+### package initialize
+``` coffeescript
+consumeAutosave: (dontSaveIf) ->
```

Comment:
Let's use ES6 for this example instead because we're trying to phase out CoffeeScript, though it's going to take a while.